spec/requests/api/v1/comments_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+RSpec.describe '/comments', type: :request do
+  let(:valid_attributes) do
+    attributes_for :comment
+  end
+
+  let(:invalid_attributes) do
+    attributes_for :comment, :invalid_short_body
+  end
+
+  let(:valid_headers) do
+    { 'ACCEPT' => 'application/json' }
+  end
+
+  let(:movie_comment) { create(:comment) }
+  let(:person_comment) { create(:comment, :for_person) }
+  let(:nested_comment) { create(:comment, :for_comment) }
+
+  describe 'POST /create' do
+    context 'with valid parameters' do
+      it 'creates a new movie Comment' do
+        expect do
+          post api_v1_movie_comments_url(movie_id: movie_comment.commentable_id),
```

Comment:
what happens when "movie_comment.commentable_id" is called?
sometimes you can check the functionality by hand)))

Reply:
![image](https://user-images.githubusercontent.com/70864999/156939481-93eb03e1-c5e4-438d-ba12-10ff2df25e69.png)
Can you specify?